README.md

```diff
@@ -6,60 +6,75 @@
 
 Reference implementation for the Connected Diagnostics API (http://dxapi.org/)
 
-## Getting Started
+## CDX Core Server. Getting Started
 
-### CDX Core Server
-
-To start developing:
+### Development environment (with Docker)
 
 1. Clone the repo.
 
-2. Install dependencies:
-	* `bundle install`.
-	* PhantomJS 1.9.8 for [Poltergeist](https://github.com/teampoltergeist/poltergeist) (development and test only)
-		* Install it in mac with: `brew install phantomjs`
-	* ImageMagick for [Paperclip](https://github.com/thoughtbot/paperclip#image-processor)
-		* Install it in mac with: `brew install imagemagick`
-	* [Redis](http://redis.io/download) is [used](https://github.com/mperham/sidekiq/wiki/Using-Redis) by [sidekiq](http://sidekiq.org/). CDX uses sidekiq as [ActiveJob](http://guides.rubyonrails.org/active_job_basics.html#backends) backend
-		* Install it in mac with: `brew install redis`
-		* you can start it with `redis-server --daemonize yes`
-	* [Elasticsearch](https://www.elastic.co/) is used as the main index for test results.
-		* We support elasticsearch versions < 2.x
-		* Install it in mac with: `brew install elasticsearch17`
+2. Install Docker.
 
-3. Setup development database: `bundle exec rake db:setup`
+3. Install `gems`:
+	```
+	$ docker-compose build
+	$ docker-compose run --rm web gem install bundler -v `<2`
+	$ docker-compose run --rm web bundle install
+	```
 
-4. Setup test database: `bundle exec rake db:test:prepare`
+4. Setup development and test databases:
+	```
+	$ docker-compose run --rm web rake db:setup
+	$ docker-compose run --rm web rake db:test:prepare
+	```
 
-5. Setup elasticsearch index template: `bundle exec rake elasticsearch:setup`
+5. Setup elasticsearch (we need the environment running):
+	```
+	# environment up
+	$ docker-compose up
 
-6. Run tests: `bundle exec rake` (this will run `rspec` and `cucumber`)
+	# setup elasticsearch index template
+	$ docker-compose run --rm web rake elasticsearch:setup
```

Comment:
I'd run a single `docker-compose run --rm web rake db:setup db:test:prepare elasticsearch:setup` as all we care about is having the environment ready to work on it.

Reply:
Fixed it. It works great!